ui/article-renderer/build.gradle

```diff
@@ -20,4 +22,6 @@ dependencies {
     implementation "com.jakewharton:butterknife:${deps.butterknife}"
 
     annotationProcessor "com.jakewharton:butterknife-compiler:${deps.butterknife}"
+
+    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
```

Comment:
we are already including constraint-layout in this module. We are using the version in the AndroidX namespace per https://developer.android.com/topic/libraries/support-library/androidx-overview